### PR TITLE
chore: add gitpod

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
- #9 

<a href="https://gitpod.io/#https://github.com/krshdev/username-checker/pull/11"></a>

### CODE REVIEW

**Review by krshdev — CHANGES_REQUESTED** [review]
Hey 👋 You need to add .gitpod.yml, please refer to the blog

**Comment by Shivansh-yadav13** [comment]
@krshdev can you please review and suggest changes if required.

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+tasks:
+  - init: |
+      npm install
+      gp sync-done installation
+    command: npm run lint
+  - init: gp sync-await installation
+    command: npm run build && npm run start
```
Comment (krshdev) [inline .gitpod.yml:7]: Hi 👋 Just one more thing is missing before we can merge this PR, you need to add prebuilds, and that's it.

**Review by krshdev — CHANGES_REQUESTED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (Shivansh-yadav13) [reply .gitpod.yml:4]: Sure 👍 I'm first time writing Gitpod file 😅

**Review by krshdev — APPROVED** [review]
Thanks for the contribution! :D

**Pull request merged**